renku_notebooks/util/gitlab_.py

```diff
@@ -70,41 +70,39 @@ def get_notebook_image(user, namespace, project, commit_sha):
     """Check if the image built by GitLab CI is ready."""
     gl_project = get_project(user, namespace, project)
 
-    image = os.getenv("NOTEBOOKS_DEFAULT_IMAGE", "renku/singleuser:latest")
+    default_image = os.getenv("NOTEBOOKS_DEFAULT_IMAGE", "renku/singleuser:latest")
 
     commit_sha_7 = commit_sha[:7]
-
-    for pipeline in gl_project.pipelines.list():
-        if pipeline.attributes["sha"] == commit_sha:
-            status = _get_job_status(pipeline, "image_build")
```

Comment:
The `_get_job_status` function can be removed.

Reply:
Done!